test/e2e/integration/frontend-test/pdf.ts

```diff
@@ -90,7 +90,8 @@ describe('PDF', () => {
     });
   });
 
-  it('should generate PDF for group step', () => {
+  // FIXME: Not working in main and needs bigger fix?
+  it.skip('should generate PDF for group step', () => {
```

Comment:
Since when? It worked 40 mins ago?

https://cloud.cypress.io/projects/y2jhp6/runs/6332/test-results/f50734d6-53ea-4e09-9916-50cb704aa5c2?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D

Reply:
Good catch. Funka ikke for meg lokalt på main da jeg testa, men funker nå lokalt både i denne branchen og i main 😮 🤷‍♀️